debian/rules

```diff
@@ -12,6 +12,17 @@ export DISTUTILS_DEBUG=1
 export DH_VERBOSE=1
 export DH_OPTIONS=-v
 
+TRUSTY_DEP = python-h5py (>= 2.2.1)
+PRECISE_DEP = python-h5py (= 2.2.1-1build3~precise03)
+
+ifeq ($(shell lsb_release --codename --short),trusty)
```

Comment:
Since `trusty` is now the default I would switch the if: `if precise: ... then ...` (and this should be backported to the engine too)